modules/custom/rest_entity_recursive/rest_entity_recursive.info.yml

```diff
@@ -0,0 +1,9 @@
+name: REST Entity Recursive
+description: Adds format to REST endpoints. The request with the format includes entity with whole references entities.
+package: REST
+type: module
+core: 8.x
+dependencies:
+  - consumer_image_styles
```

Comment:
Can we drop consumer_image_styles dependency? Not everyone will need it.

Reply:
Yes we can, but we need to remove ImageItemNormalizer from this module. Actually we don`t need it in this module.